### PR TITLE
opening header amended to include tabs

### PR DESCRIPTION
**Pull request by KimberleyCook**

### CODE REVIEW

**Comment by jonsherrard** [comment]
Very nice!

**Comment by jonsherrard** [comment]
@KimberleyCook I take it back! Can you re-raise this into master and then it's good to merge, just let me know when you're done and I'll tag it up.

**Comment by ecoad** [comment]
(Bonus points for a screenshot for this kind of PR)

File: custom-fields/basic/header.php [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
   'fields' => array (
     array (
+      'key' => 'header_basic_details_tab',
+      'label' => 'Basic Details',
+      'name' => '',
+      'type' => 'tab',
+      'instructions' => '',
+      'required' => 0,
+      'conditional_logic' => 0,
+      'wrapper' => array (
```
Comment (ecoad) [inline custom-fields/basic/header.php]: Can you delete the unused/default lines of code here?

**Comment by ecoad** [comment]
In `header.php` there's plenty of unused ACF declarations can you clear these up please?

**Pull request merged**